include/data_management/data_source/mysql_feature_manager.h

```diff
@@ -304,10 +314,10 @@ class SQLFeatureManager
         return status;
     }
 
-private:
+public:
     internal::SQLFetchBufferPtr _fetchBuffer;
     BlockDescriptor<DAAL_DATA_TYPE> _currentRowBlock;
-    services::SharedPtr<services::ErrorCollection> _errors;
+    services::SharedPtr<services::ErrorCollection> _errors = services::SharedPtr<services::ErrorCollection>(new services::ErrorCollection);
```

Comment:
This works starting from C++11 only. In header files DAAL supports C++ starting from C++03 version. So, you should replace this.